### DISCUSSION TITLE
Welcome to docker-chromium-armhf Discussions!

### DISCUSSION THREAD

**Discussion by HenningThiemann** [HenningThiemann/docker-chromium-armhf#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Boteium** [comment]
Is this repo still being maintained ? There are pull requests pending for approval for some time. Can you take a look ?

**Reply by HenningThiemann** [reply]
Hi! 
The manjaro arm guys switched it over to their gitlab (https://gitlab.manjaro.org/manjaro-arm/packages/community/chromium-docker).  As a result, I've stopped maintaining this repo. 

I can merge your PRs anyway, if you want me to do this. But as I am no expert reg. PulseAudio, I will not be able to check these and need to rely on your judgement, whether this are non-breaking PRs.

**Reply by Boteium** [reply]
is spikerguy currently the maintainer ? or is spikerguy only the maintainer of manjaro arm's version of this package?

If spikerguy is the maintainer and this repo is not being maintained, perhaps you should archive this repo and let people know that "https://github.com/spikerguy/docker-chromium-armhf/commits/master" has became the upstream.

Or, if spikerguy is only maintaining manjaro's version of this package. I think you are still the upstream.

Btw, I personally don't use manjaro arm. So, my instinct is that this repo is the upstream. I guess most people cloning this repo are not using manjaro either because manjaro user can just grabbed the pre-built package.
